libs/deposits/src/lib/deposit-form.tsx

```diff
@@ -152,7 +151,11 @@ export const DepositForm = ({
         <Controller
           control={control}
           name="asset"
-          rules={{ validate: { required } }}
+          rules={{
+            validate: {
+              required: (value: string) => !!selectedAsset || required(value),
```

Comment:
question (blocking): Why not just let it be validated normally here using defaultValues?

Reply:
updated